update docker image tagging descriptions

taz-pega-work opened the pull request:
review documentation for docker image tagging and clarify latest changes

dcasavant commented on docs/Deploying-Pega-on-PKS.md:
```diff
@@ -423,7 +423,7 @@ tier:
       # Enter the domain name to access web nodes via a load balancer.
       #  e.g. web.mypega.example.com
-      domain: "**pks.web.dev.pega.io**"
+      domain: "**TKGI.web.dev.pega.io**"
```
Some of these have switched from lowercase (pks) to uppercase (TKGI).  Is that intentional or a find/replace artifact?

taz-pega-work replied:
this is due to find/replace

dcasavant commented on docs/Deploying-Pega-on-PKS.md:
```diff
@@ -54,23 +54,23 @@ The following account, resources, and application versions are required for use
 - kubectl – the Kubernetes command-line tool that you use to connect to and manage your Kubernetes resources.
 
-- PKS CLI - the PKS command-line tool that you use to manage and communicate with your PKS cluster.
+- Cloud Foundry CLI - the Cloud Foundry command-line tool that you use to manage and communicate with your TKGI cluster.
```
Is Cloud Foundry correct?  That is a different technology.

taz-pega-work replied:
updated the references back to use pks cli...there's no more Cloud Foundry or cf CLI mentions.

dcasavant commented on docs/Deploying-Pega-on-PKS.md:
```diff
@@ -1,16 +1,16 @@
-Deploying Pega Platform on a PKS cluster
+Deploying Pega Platform on a TKGI cluster
```
Is it worth noting in the beginning of this document that TKGI was formerly known as PKS?

taz-pega-work replied:
good thought...I added statement at the beginning: Deploy Pega Platform™ on a VMware Tanzu Kubernetes Grid Integrated Edition (TKGI) cluster using a PostgreSQL database you configure in Google Cloud Platform (GCP). TKGI was formerly knows Pivotal Container Service (PKS).

dcasavant commented on charts/pega/README.md:
```diff
@@ -12,7 +12,7 @@ k8s         | Open-source Kubernetes
 eks         | Amazon Elastic Kubernetes Service (EKS)
 gke         | Google Kubernetes Engine (GKE)
-pks         | Pivotal Container Service (PKS)
+pks         | VMware Tanzu Kubernetes Grid Integrated Edition (TKGI)
```
Should we update the value so that `pks || tkgi` results in the same deployment configuration?

taz-pega-work replied:
I don't know. I updated both sections. Let me know if this makes sense now.

dcasavant commented on charts/pega/README.md:
```diff
@@ -265,7 +265,7 @@ Provider  | Kubernetes Secrets | Cloud SSL management service
 AKS       | Supported          | None
 EKS       | Not supported      | Manage certificate using Amazon Certification Manager and use ssl_annotation - see example for details.
-PKS       | Supported          | None
+TKGI       | Supported          | None
```
According to the earlier section, the provider *value* is still `pks`, while the *target* is TKGI... Maybe we need to unify this a bit.

taz-pega-work replied:
the text is updated; I don't know if anyone has updated with provider text, so I can make further changes if that is required.

dcasavant reviewed (commented):
Comments inline

taz-pega-work commented:
Updated Docker image tagging and updated support for the change from Pivotal Container Service (PKS) to VMware Tanzu Kubernetes Grid Integrated Edition (TKGI)
